backend/api/src/main/java/com/virtuslab/gitmachete/backend/hooks/OnExecutionTimeout.java

```diff
@@ -1,5 +1,5 @@
 package com.virtuslab.gitmachete.backend.hooks;
 
 public enum OnExecutionTimeout {
-  THROW, RETURN_NULL
+  RETURN_NULL, THROW_EXCEPTION
```

Comment:
iirc in c++ one is able to throw anything (e.g. `int`) but in java this specification seems to be redundant indeed